### ISSUE TITLE
Native "HTTPClient.HTTPClient" was not found

### ISSUE THREAD

**Issue by freakexeuLow** [Sarrus1/shavit-discord#1]
hey Sarrus, i test your new version and i get this error
```
L 02/23/2021 - 23:33:41: SourceMod error session started
L 02/23/2021 - 23:33:41: Info (map "bhop_arcane_classic") (file "/home/user2-1/server/csgo/csgo/addons/sourcemod/logs/errors_20210223.log")
L 02/23/2021 - 23:33:41: [SM] Unable to load plugin "shavit-discord.smx": Native "HTTPClient.HTTPClient" was not found
L 02/23/2021 - 23:33:46: Error log file session closed.
```

**Comment by Sarrus1** [comment]
Hello,

Can you type this in your server's console `sm exts list` and paste the results here please?

**Comment by freakexeuLow** [comment]
> Hello,
> 
> Can you type this in your server's console `sm exts list` and paste the results here please?

sm exts list
```
[SM] Displaying 21 extensions:
[01] Automatic Updater (1.10.0.6502): Updates SourceMod gamedata files
[02] Webternet (1.10.0.6502): Extension for interacting with URLs
[03] Console Cleaner (1.1.0): Console warning suppressor
[04] Accelerator (2.5.0): SRCDS Crash Handler
[05] CS Tools (1.10.0.6502): CS extended functionality
[06] BinTools (1.10.0.6502): Low-level C/C++ Calling API
[07] SDK Tools (1.10.0.6502): Source SDK Tools
[08] <OPTIONAL> file "rip.ext.so": /home/user2-1/server/csgo/csgo/addons/sourcemod/extensions/rip.ext.so: cannot open shared object file: No such file or directory
[09] SMJansson (2.3.1/3): JSON parser/writer
[10] Client Preferences (1.10.0.6502): Saves client preference settings
[11] SQLite (1.10.0.6502): SQLite Driver
[12] Regex (1.10.0.6502): Provides regex natives for plugins
[13] SDK Hooks (1.10.0.6502): Source SDK Hooks
[14] DHooks (2.2.0-detours15): Dynamic Hooks
[15] <OPTIONAL> file "sendproxy.ext.so": /home/user2-1/server/csgo/csgo/addons/sourcemod/extensions/sendproxy.ext.so: cannot open shared object file: No such file or directory
[16] GeoIP (1.10.0.6494): Geographical IP information
[17] Top Menus (1.10.0.6502): Creates sorted nested menus
[18] SteamWorks Extension (1.2.3): Exposes SteamWorks functions to Developers
[19] PTaH (1.1.2 Test): Additional CS:GO Hooks and Natives
[20] Socket (3.0.2): Socket extension for SourceMod
[21] MySQL-DBI (1.10.0.6502): MySQL driver implementation for DBI
```

**Comment by Sarrus1** [comment]
Ok, try to install [this](https://github.com/ErikMinekus/sm-ripext/releases/tag/1.2.3) extension to see if that fixes the issue.

**Comment by freakexeuLow** [comment]
> Ok, try to install [this](https://github.com/ErikMinekus/sm-ripext/releases/tag/1.2.3) extension to see if that fixes the issue.

so you have to use ripext after all, i thought if you don't use it because "(optional if you don't want the profile picture)"^^

**Comment by Sarrus1** [comment]
You shouldn't have to use it, but I guess I messed up the dependencies :/
I will mark it as required until I fix this.

Did it solve your issue?

**Comment by freakexeuLow** [comment]
> You shouldn't have to use it, but I guess I messed up the dependencies :/
> I will mark it as required until I fix this.
> 
> Did it solve your issue?

The error is gone, actually did not want to use ripext and now waiting until you fixed the problem 👍🏼

**Comment by Sarrus1** [comment]
If you don't set an api key in your cfg file, the extension won't run.

I will comment here when the issue is fixed 👍

**Comment by freakexeuLow** [comment]
> If you don't set an api key in your cfg file, the extension won't run.
> 
> I will comment here when the issue is fixed 👍

that is also true again, nevertheless it is on it and okay

**Comment by Sarrus1** [comment]
Try this commit 382f495a96d6f8bbe0bb7bf376f54137f75ccaa7

**Comment by freakexeuLow** [comment]
> Try this commit [382f495](https://github.com/Sarrus1/shavit-discord/commit/382f495a96d6f8bbe0bb7bf376f54137f75ccaa7)

so if this is to run without ripext, the error is the same Sarrus

**Comment by Sarrus1** [comment]
Hey, I'm back, managed to fix the issue in 68a719854525dc5a77ce1f309be750f645530955

Let me know if it works for you as well 👍
